config/util/helpers.go

```diff
@@ -41,3 +41,12 @@ func IsTrue(b *bool) bool {
 func IsFalse(b *bool) bool {
 	return b != nil && !*b
 }
+
+func StrSliceContains(slice []string, s string) bool {
```

Comment:
This is a bit odd, since this is public API (immutable after it's included in a release) and nothing in the `config` package needs the function.  I'd consider putting the function in `internal`.

Reply:
Ouff, didn't realize that package was public. Thanks!